l10n_eu_nace/partner.py

```diff
@@ -19,22 +19,22 @@
 #
 ##############################################################################
 
-from openerp.osv import osv
+from openerp import models
 
 
-class PartnerCategory(osv.Model):
+class PartnerCategory(models.Model):
     """Let users search on code without a dot"""
     _inherit = 'res.partner.category'
 
-    def name_search(self, cr, uid, name, args=None, operator='ilike',
-                    context=None, limit=80):
+    def _name_search(self, cr, user, name='', args=None, operator='ilike',
```

Comment:
As you are already passing this to new API, change the signature removing cr and user, and put @api.model decorator.